### DISCUSSION TITLE
~/.tridactylrc not loading

### DISCUSSION THREAD

**Discussion by johnknl** [tridactyl/tridactyl#4566] [Q&A] [answered]
I did a fresh install on my PC, lost all my FF settings, reinstalled tridactyl, copied over the old file to ~/.tridactylrc but it's not being loaded. Even manually explicitly doing `:source ~/.tridactylrc` doesn't change anything.

What else can I try?

**Comment by bovine3dom** [comment, marked as answer]
Is `:native` installed and working? If it isn't you need to install it.

If it is, usually there is another RC file that is being read instead - e.g. .config/tridactyl/tridactylrc

**Reply by johnknl** [reply]
Did not install that. It's been so long since I originally installed tridactyl that I just forgot. Parusing the README it didn't immediately jump out at me but in hindsight..

Thanks and sorry for asking obvious questions ;)

**Comment by bovine3dom** [comment]
Not at all - we really need to improve the lack of error messages :)
